shop/src/main/java/ro/msg/learning/shop/controller/test/TestDataBaseController.java

```diff
@@ -0,0 +1,31 @@
+package ro.msg.learning.shop.controller.test;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ro.msg.learning.shop.service.ManipulateDatabaseService;
+import ro.msg.learning.shop.utils.Status;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/test")
+@Profile("test")
+public class TestDataBaseController {
```

Comment:
kihihi. This is unorthodox but imaginative. The population of the db can be done as well from the testing classes and be automated in the beofre each and after methods. They should not require manual usage of postman to make your db. Also, nice that you tried, but at unit testing level we dont check what happens to the db. That is a step further, in the integration tests. Its a Strutocamila